static/js/stream_list.js

```diff
@@ -461,9 +461,10 @@ exports.initialize = function () {
         if (overlays.is_active()) {
             ui_util.change_tab_to('#home');
         }
-        var stream = $(e.target).parents('li').attr('data-name');
+        var stream_id = $(e.target).parents('li').attr('data-stream-id');
+        var stream = stream_data.get_sub_by_id(stream_id);
```

Comment:
I think we should stick with the `sub` naming convention here.   I don't really like it, but we use `sub` for the structs and often use `stream` for the stream name.